ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJkZWZhdWx0IiwibG9jYWxlIjoiaXRfSVQiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUy8%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQyIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEzNzI1MzY1NTIuMTc0NTQ0OTMzOCJ9LCJhbm9uSWQiOiI5NTNkZjRkMjcyMDA3MzkwIn0sInBlcmZvcm1hbmNlIjp7fSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6e30sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiNjE5NzUxMDYxOTg2NDk2NyJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55b1110a7cea <unknown>
#1 0x55b110b585f0 <unknown>
#2 0x55b110b3ee86 <unknown>
#3 0x55b110b3d79a <unknown>
#4 0x55b110b3deff <unknown>
#5 0x55b110b3de54 <unknown>
#6 0x55b110b5c7ff <unknown>
#7 0x55b110c019aa <unknown>
#8 0x55b110bcf662 <unknown>
#9 0x55b110bf5660 <unknown>
#10 0x55b110bcf433 <unknown>
#11 0x55b110b9bea3 <unknown>
#12 0x55b110b9cb01 <unknown>
#13 0x55b11106cb3b <unknown>
#14 0x55b111070a21 <unknown>
#15 0x55b111053c32 <unknown>
#16 0x55b111071594 <unknown>
#17 0x55b111037eef <unknown>
#18 0x55b111095d98 <unknown>
#19 0x55b111095f76 <unknown>
#20 0x55b1110a6b36 <unknown>
#21 0x7f7d3249caa4 <unknown>
#22 0x7f7d32529c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gc2VhcmNoLXJlc3VsdHMiLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEzNzI1MzY1NTIuMTc0NTQ0OTMzOCJ9LCJhbm9uSWQiOiI5NTNkZjRkMjcyMDA3MzkwIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUy8%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQyIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIyOTkzMjA5NTgwMTQxMzk2IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55b1110a7cea <unknown>
#1 0x55b110b585f0 <unknown>
#2 0x55b110b3ee86 <unknown>
#3 0x55b110b3d79a <unknown>
#4 0x55b110b3deff <unknown>
#5 0x55b110b3de54 <unknown>
#6 0x55b110b5c7ff <unknown>
#7 0x55b110c019aa <unknown>
#8 0x55b110bcf662 <unknown>
#9 0x55b110bf5660 <unknown>
#10 0x55b110bcf433 <unknown>
#11 0x55b110b9bea3 <unknown>
#12 0x55b110b9cb01 <unknown>
#13 0x55b11106cb3b <unknown>
#14 0x55b111070a21 <unknown>
#15 0x55b111053c32 <unknown>
#16 0x55b111071594 <unknown>
#17 0x55b111037eef <unknown>
#18 0x55b111095d98 <unknown>
#19 0x55b111095f76 <unknown>
#20 0x55b1110a6b36 <unknown>
#21 0x7f7d3249caa4 <unknown>
#22 0x7f7d32529c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJTZWFyY2ggUmVzdWx0IFBhZ2UgLSBDYXJzIiwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMzcyNTM2NTUyLjE3NDU0NDkzMzgifSwiYW5vbklkIjoiOTUzZGY0ZDI3MjAwNzM5MCJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3NlYXJjaC1yZXN1bHRzLmh0bWwvdmVoaWNsZUNhdGVnb3J5LW5ldy1wYXNzZW5nZXItY2Fycy9icmFuZC1NZXJjZWRlcy1CZW56L21vZGVsSWRlbnRpZmllci1FUUVfQ0xBU1M%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQyIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI1MTg5ODYyMzQ2MDc5MTgyIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55b1110a7cea <unknown>
#1 0x55b110b585f0 <unknown>
#2 0x55b110b3ee86 <unknown>
#3 0x55b110b3d79a <unknown>
#4 0x55b110b3deff <unknown>
#5 0x55b110b3de54 <unknown>
#6 0x55b110b5c7ff <unknown>
#7 0x55b110c019aa <unknown>
#8 0x55b110bcf662 <unknown>
#9 0x55b110bf5660 <unknown>
#10 0x55b110bcf433 <unknown>
#11 0x55b110b9bea3 <unknown>
#12 0x55b110b9cb01 <unknown>
#13 0x55b11106cb3b <unknown>
#14 0x55b111070a21 <unknown>
#15 0x55b111053c32 <unknown>
#16 0x55b111071594 <unknown>
#17 0x55b111037eef <unknown>
#18 0x55b111095d98 <unknown>
#19 0x55b111095f76 <unknown>
#20 0x55b1110a6b36 <unknown>
#21 0x7f7d3249caa4 <unknown>
#22 0x7f7d32529c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJzZWFyY2hfcmVzdWx0IiwibG9jYWxlIjoiaXRfSVQiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUz9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9U1VWJmVtaGZ1ZWxUeXBlPUVMRUNUUklDIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTM3MjUzNjU1Mi4xNzQ1NDQ5MzM4In0sImFub25JZCI6Ijk1M2RmNGQyNzIwMDczOTAifSwicGVyZm9ybWFuY2UiOnt9LCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9LCJjYXRhbG9nIjp7fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIzMTEwNTQ4NzczMDcwNzU1In0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55b1110a7cea <unknown>
#1 0x55b110b585f0 <unknown>
#2 0x55b110b3ee86 <unknown>
#3 0x55b110b3d79a <unknown>
#4 0x55b110b3deff <unknown>
#5 0x55b110b3de54 <unknown>
#6 0x55b110b5c7ff <unknown>
#7 0x55b110c019aa <unknown>
#8 0x55b110bcf662 <unknown>
#9 0x55b110bf5660 <unknown>
#10 0x55b110bcf433 <unknown>
#11 0x55b110b9bea3 <unknown>
#12 0x55b110b9cb01 <unknown>
#13 0x55b11106cb3b <unknown>
#14 0x55b111070a21 <unknown>
#15 0x55b111053c32 <unknown>
#16 0x55b111071594 <unknown>
#17 0x55b111037eef <unknown>
#18 0x55b111095d98 <unknown>
#19 0x55b111095f76 <unknown>
#20 0x55b1110a6b36 <unknown>
#21 0x7f7d3249caa4 <unknown>
#22 0x7f7d32529c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gcHJvZHVjdCIsInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTM3MjUzNjU1Mi4xNzQ1NDQ5MzM4In0sImFub25JZCI6Ijk1M2RmNGQyNzIwMDczOTAifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9idXkvbmV3LWNhci9wcm9kdWN0Lmh0bWwvRVFFLTMwMC1TVVZfMDM1NDM0Mjc0MF9JVF8zNTRhZTJmNSIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIxMDEwNzgxMDUyMjQyMTAzNSIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55b1110a7cea <unknown>
#1 0x55b110b585f0 <unknown>
#2 0x55b110b3ee86 <unknown>
#3 0x55b110b3d79a <unknown>
#4 0x55b110b3deff <unknown>
#5 0x55b110b3de54 <unknown>
#6 0x55b110b5c7ff <unknown>
#7 0x55b110c019aa <unknown>
#8 0x55b110bcf662 <unknown>
#9 0x55b110bf5660 <unknown>
#10 0x55b110bcf433 <unknown>
#11 0x55b110b9bea3 <unknown>
#12 0x55b110b9cb01 <unknown>
#13 0x55b11106cb3b <unknown>
#14 0x55b111070a21 <unknown>
#15 0x55b111053c32 <unknown>
#16 0x55b111071594 <unknown>
#17 0x55b111037eef <unknown>
#18 0x55b111095d98 <unknown>
#19 0x55b111095f76 <unknown>
#20 0x55b1110a6b36 <unknown>
#21 0x7f7d3249caa4 <unknown>
#22 0x7f7d32529c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:111 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/Last Seen PDP_attempt_1.png'
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6251fcd910>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9619d66ce8afa3248b6ab5637b3e82fe
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6251fcd250>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9619d66ce8afa3248b6ab5637b3e82fe
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6251fce250>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9619d66ce8afa3248b6ab5637b3e82fe
ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gc2VhcmNoLXJlc3VsdHMiLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwNTY3Mjg3NzMuMTc0NTQ0OTM3NSJ9LCJhbm9uSWQiOiJmZmJiYTZmOTgyODk0NmY5In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUy8%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQyIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI0ODI4NjU5NDQ2NzMxMDUzIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJkZWZhdWx0IiwibG9jYWxlIjoiaXRfSVQiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUy8%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQyIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwNTY3Mjg3NzMuMTc0NTQ0OTM3NSJ9LCJhbm9uSWQiOiJmZmJiYTZmOTgyODk0NmY5In0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnt9LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjIwNTI1NjY3NTY3ODQ2OTUifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJzZWFyY2hfcmVzdWx0IiwibG9jYWxlIjoiaXRfSVQiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUz9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9U1VWJmVtaGZ1ZWxUeXBlPUVMRUNUUklDIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTA1NjcyODc3My4xNzQ1NDQ5Mzc1In0sImFub25JZCI6ImZmYmJhNmY5ODI4OTQ2ZjkifSwicGVyZm9ybWFuY2UiOnt9LCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9LCJjYXRhbG9nIjp7fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIwNDk3NTUzNjQ1NDg5MjAwNDUifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJTZWFyY2ggUmVzdWx0IFBhZ2UgLSBDYXJzIiwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMDU2NzI4NzczLjE3NDU0NDkzNzUifSwiYW5vbklkIjoiZmZiYmE2Zjk4Mjg5NDZmOSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3NlYXJjaC1yZXN1bHRzLmh0bWwvdmVoaWNsZUNhdGVnb3J5LW5ldy1wYXNzZW5nZXItY2Fycy9icmFuZC1NZXJjZWRlcy1CZW56L21vZGVsSWRlbnRpZmllci1FUUVfQ0xBU1M%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQyIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI4MDkyNTQwMDc1MjI2NjE1IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFNSUE1vZGVsRmlsdGVyIjoie1wiZGNwX2xhc3RfbW9kZWxmaWx0ZXJcIjpcImVxZS1jbGFzc1wiLFwiZGNwX2xhc3RfYm9keXR5cGVmaWx0ZXJcIjpcInN1dlwiLFwiZGNwX2xhc3Rfc2VhcmNoX3VybFwiOlwiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3NlYXJjaC1yZXN1bHRzLmh0bWwvdmVoaWNsZUNhdGVnb3J5LW5ldy1wYXNzZW5nZXItY2Fycy9icmFuZC1NZXJjZWRlcy1CZW56L21vZGVsSWRlbnRpZmllci1FUUVfQ0xBU1M%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPVNVViZlbWhmdWVsVHlwZT1FTEVDVFJJQ1wiLFwidGltZXN0YW1wXCI6MTc0NTQ0OTM4MDQyNSxcImV4cGlyYXRpb25cIjoxMjB9In19LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwNTY3Mjg3NzMuMTc0NTQ0OTM3NSJ9LCJhbm9uSWQiOiJmZmJiYTZmOTgyODk0NmY5In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUVRRV9DTEFTUz9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9U1VWJmVtaGZ1ZWxUeXBlPUVMRUNUUklDIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sIl90b29sc0V2ZW50TGlua0lkIjoiOTgzODUwMTQ0NzE3NzAxOSIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gcHJvZHVjdCIsInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTA1NjcyODc3My4xNzQ1NDQ5Mzc1In0sImFub25JZCI6ImZmYmJhNmY5ODI4OTQ2ZjkifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9idXkvbmV3LWNhci9wcm9kdWN0Lmh0bWwvRVFFLTMwMC1TVVZfMDM1NDM0Mjc0MF9JVF8zNTRhZTJmNSIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI5MzIwNDE1Mzk4NjUxMDE2IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IERldGFpbCBQYWdlIC0gMjk0NiIsInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTA1NjcyODc3My4xNzQ1NDQ5Mzc1In0sImFub25JZCI6ImZmYmJhNmY5ODI4OTQ2ZjkifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9idXkvbmV3LWNhci9wcm9kdWN0Lmh0bWwvRVFFLTMwMC1TVVZfMDM1NDM0Mjc0MF9JVF8zNTRhZTJmNSIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxLCJzZGtMb2FkVGltZSI6M30sIl90b29sc0V2ZW50TGlua0lkIjoiMTE2MjEzMTIyNjM4MzQ5NiIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJwcm9kdWN0X2RldGFpbCIsImxvY2FsZSI6Iml0X0lUIiwiY29udGVudFpvbmVzIjpbXSwidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3Byb2R1Y3QuaHRtbC9FUUUtMzAwLVNVVl8wMzU0MzQyNzQwX0lUXzM1NGFlMmY1IiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTA1NjcyODc3My4xNzQ1NDQ5Mzc1In0sImFub25JZCI6ImZmYmJhNmY5ODI4OTQ2ZjkifSwicGVyZm9ybWFuY2UiOnsiZG9tTG9hZFRpbWUiOjQ3MTAsInBhZ2VMb2FkVGltZSI6NDcyMiwic2RrUGFyc2VUaW1lIjoxLCJzZGtMb2FkVGltZSI6M30sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnt9LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjE0NjM4NDE1ODA0MjQxMjM0In0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IERldGFpbCBQYWdlIiwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMDU2NzI4NzczLjE3NDU0NDkzNzUifSwiYW5vbklkIjoiZmZiYmE2Zjk4Mjg5NDZmOSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3Byb2R1Y3QuaHRtbC9FUUUtMzAwLVNVVl8wMzU0MzQyNzQwX0lUXzM1NGFlMmY1IiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJwZXJmb3JtYW5jZSI6eyJkb21Mb2FkVGltZSI6NDcxMCwicGFnZUxvYWRUaW1lIjo0NzIyLCJzZGtQYXJzZVRpbWUiOjEsInNka0xvYWRUaW1lIjozfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI0MTAxODA3NDQxMzkwMDAyIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x55d2c76d7cea <unknown>
#1 0x55d2c71885f0 <unknown>
#2 0x55d2c716ee86 <unknown>
#3 0x55d2c716d79a <unknown>
#4 0x55d2c716deff <unknown>
#5 0x55d2c716de54 <unknown>
#6 0x55d2c718c7ff <unknown>
#7 0x55d2c72319aa <unknown>
#8 0x55d2c71ff662 <unknown>
#9 0x55d2c7225660 <unknown>
#10 0x55d2c71ff433 <unknown>
#11 0x55d2c71cbea3 <unknown>
#12 0x55d2c71ccb01 <unknown>
#13 0x55d2c769cb3b <unknown>
#14 0x55d2c76a0a21 <unknown>
#15 0x55d2c7683c32 <unknown>
#16 0x55d2c76a1594 <unknown>
#17 0x55d2c7667eef <unknown>
#18 0x55d2c76c5d98 <unknown>
#19 0x55d2c76c5f76 <unknown>
#20 0x55d2c76d6b36 <unknown>
#21 0x7f7e79c9caa4 <unknown>
#22 0x7f7e79d29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/Last Seen PDP_attempt_2.png'
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6252ed9850>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9619d66ce8afa3248b6ab5637b3e82fe
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6252ed9910>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9619d66ce8afa3248b6ab5637b3e82fe
WARNING  urllib3.connectionpool:connectionpool.py:868 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6252edb810>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9619d66ce8afa3248b6ab5637b3e82fe